admin/src/scenes/classe/header/VerifClassButton.tsx

```diff
@@ -41,24 +41,23 @@ export default function VerifClassButton({ classe, setClasse, isLoading, setLoad
   };
 
   const verifyClasse = async () => {
-    try {
-      setLoading(true);
-
-      const { ok, code, data } = await api.put(`/cle/classe/${classe?._id}/verify`, classe);
-
-      if (!ok) {
-        toastr.error("Oups, une erreur est survenue lors de la vérification de la classe", translate(code));
-        return setLoading(false);
-      } else {
+    setLoading(true);
+    await apiv2
+      .post<ClassesRoutes["GetOne"]["response"]["data"]>(`/classe/${classe?._id}/verify`, classe)
+      .then((updatedClasse) => {
```

Comment:
tu aimes les `then`...
await est plus lisible quand même

Reply:
```
 try {
      ...
    } catch (error: unknown) {
        if (error instanceof AxiosError<HttpError> && error.response?.status === 422) {
             ....
        } else instanceof Error {
            ...
        } else {
           // yolo
        }
    }
```
c'est cette logique qu'il faudrait déporté dans le wrapper pour throw nos erreurs à nous https://stackoverflow.com/questions/55065742/implementing-instanceof-checks-for-custom-typescript-error-instances

Reply:
hum... à réfléchir...